lyricsgenius/api/base.py

```diff
@@ -3,7 +3,7 @@
 import time
 
 
-class Sender:
+class Sender(object):
```

Comment:
All base classes inherit `object` in Python 3 regardless of typing `(object)` or not. So I left it out.